cadasta/gui/tools/helper/content/create_project_help.py

```diff
@@ -0,0 +1,83 @@
+# coding=utf-8
+"""Help for create download."""
+
+from cadasta.utilities.i18n import tr
+from extras import messaging as m
+from extras.messaging import styles
+
+INFO_STYLE = styles.INFO_STYLE
+
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__date__ = '03/01/17'
+
+
+def create_project_help():
+    """Help message for Create Project.
+
+    :returns: A message object containing helpful information.
+    :rtype: messaging.message.Message
+    """
+
+    message = m.Message()
+    message.add(heading())
+    message.add(content())
+    return message
+
+
+def heading():
+    """Helper method that returns just the header.
+
+    This method was added so that the text could be reused in the
+    other contexts.
+
+    :returns: A heading object.
+    :rtype: safe.messaging.heading.Heading
+    """
+    message = m.Heading(tr('Create Project Help'), **INFO_STYLE)
+    return message
+
+
+def content():
+    """Helper method that returns just the content.
+
+    This method was added so that the text could be reused in the
+    other contexts.
+
+    :returns: A message object without brand element.
+    :rtype: safe.messaging.message.Message
+    """
+
+    message = m.Message()
+
+    message.add(m.Paragraph(tr(
+        'Create Project is one of main wizard that will help you to create'
+        'project in qgis and update it to Cadasta with easy steps.'
+        '<b>This wizard</b>')))
```

Comment:
I'm not a messaging expert, but I think we shouldn't use HTML in the message itself. We can export the message as markdown, json ...
There is also some html tags below in the code.